api/report_server.thrift

```diff
@@ -303,66 +300,36 @@ service codeCheckerDBAccess {
+  // contain exactly one directory which has a "reports" and an optional "root"
+  // sub-folder. The former one is the output of "CodeChecker analyze" command
+  // and the latter one contains the source files on absolute paths starting as
+  // if "root" was the "/" directory. The source files are not necessary to be
+  // wrapped in the ZIP file (see necessaryFileContents() function).
+  // The "version" parameter is the used CodeChecker version which checked this
+  // run.
+  // The "force" parameter removes existing analysis results for a run.
+  i64 storeZip(
+               1: string run_name,
+               2: string version,
+               3: string zipfile,
+               4: bool force)
+               throws (1: shared.RequestFailed requestError),
 
   bool replaceConfigInfo(
                      1: i64 run_id,
                      2: shared.CheckerConfigList values)
                      throws (1: shared.RequestFailed requestError),
 
-  # the map contains a path and a comment (can be empty)
-  bool addSkipPath(
-                   1: i64 run_id,
-                   2: map<string, string> paths)
-                   throws (1: shared.RequestFailed requestError),
-
-
-  // The next few following functions must be called via the same connection.
-  // =============================================================
-  i64  addReport(
-                 1: i64 run_id,
-                 2: i64 file_id,
-                 3: string bug_hash,
-                 4: string checker_message,
-                 5: shared.BugPath bugpath,
-                 6: shared.BugPathEvents events,
-                 7: string checker_id,
-                 8: string checker_cat,
-                 9: string bug_type,
-                 10: shared.Severity severity)
-                 throws (1: shared.RequestFailed requestError),
-
-
-  // * If (filepath, content_hash) is in the DB return (existing fileId, false).
-  // * If only the content_hash matches, a (new fileId, false) is returned.
-  // * If the filepath matches, but content_hash not, a (new file_id, true) is returned.
-  // * If none of them matches a (new file_id, true) is returned.
-  NeedFileResult needFileContent(
-                                 1: string filepath,
-                                 2: string content_hash)
-                                 throws (1: shared.RequestFailed requestError),
-
-  bool addFileContent(
-                      1: string content_hash,
-                      2: string file_content,
-                      3: optional Encoding encoding)
-                      throws (1: shared.RequestFailed requestError),
-
-  bool finishCheckerRun(1: i64 run_id)
-                        throws (1: shared.RequestFailed requestError),
-
-  bool setRunDuration(1: i64 run_id,
-                      2: i64 duration)
-                      throws (1: shared.RequestFailed requestError),
-
   bool stopServer()
```

Comment:
This method is absolutely not used anywhere in the code. We should remove it too, if we are already throwing out a big hunk here.